core/server/data/migrations/versions/3.33/01-add-email-recipients-tables.js

```diff
@@ -0,0 +1,32 @@
+const logging = require('../../../../../shared/logging');
+const commands = require('../../../schema').commands;
+
+const TABLES = ['email_batches', 'email_recipients'];
+
+module.exports = {
+    async up({connection}) {
+        return Promise.each(TABLES, async (table) => {
+            const tableExists = await connection.schema.hasTable(table);
+
+            if (tableExists) {
+                return logging.warn(`Adding table: ${table}`);
+            }
+
+            logging.info(`Adding table: ${table}`);
+            return commands.createTable(table, connection);
+        });
+    },
+
+    async down({connection}) {
+        return Promise.each(TABLES.reverse(), async (table) => {
```

Comment:
`.reverse()` is needed here so we don't get tripped up on foreign key constraints

Reply:
If this needs calling out in the PR, then it probably needs to either be a comment in the code, or the code needs to be explicit 😬

Reply:
👍 updated